lib/PuppeteerSharp/Browser.cs

```diff
@@ -82,6 +89,12 @@ internal void ChangeTarget(TargetInfo targetInfo)
             return version.product.ToString();
         }
 
+        public Task DisconnectAsync()
```

Comment:
Do we need to make this Async?

Reply:
Nope, was thinking about consistency, but it doesn't need to be async!